courses/free-courses-ne.md

```diff
@@ -34,7 +35,9 @@
 * [SQL Full Course In Nepali](https://www.youtube.com/watch?v=Lt52pYaoSR8&list=PL2OJkQtHPRicxyldFGNJRRG4WwNe0Kjqe&index=2) - Technology Channel
 
 
-#### C
+#### Svelte
+
+* [Svelte - Crash Course](https://www.udemy.com/cart/subscribe/course/5049082) - Demian Kostelny, Udemy
 
```

Comment:
you removed `#### C` which is required here. also always give two newline(linebreak) between to section.